Makefile

```diff
@@ -118,7 +119,7 @@ else
 exes: $(EXES)
 
 $(EXES):
-	CGO_ENABLED=0 go build $(GO_FLAGS) -o $@ ./$(@D)
+	CGO_ENABLED=0 GOOS=$(GOOS) go build $(GO_FLAGS) -o $@ ./$(@D)
```

Comment:
why not just `export GOOS=windows` before running `make` then you don't need these two lines.